cmd/bridge/main.go

```diff
@@ -123,6 +123,7 @@ func main() {
 
 	fDevCatalogCategories := fs.String("developer-catalog-categories", "", "Allow catalog categories customization. (JSON as string)")
 	fUserSettingsLocation := fs.String("user-settings-location", "configmap", "DEV ONLY. Define where the user settings should be stored. (configmap | localstorage).")
+	fAddPageJSON := fs.String("add-page-json", "", "DEV ONLY. Allow add page customization. (JSON as string)")
```

Comment:
Would be good to align on a pattern here, with the `JSON` && `-json` suffix.
Check https://github.com/openshift/console/pull/8502#discussion_r609998847
We should be consistent here.

Reply:
Ack. Removed `JSON` and `-json` suffixes